src/core/qgsmapsettings.h

```diff
@@ -141,6 +141,27 @@ class CORE_EXPORT QgsMapSettings
     //! Get units of map's geographical coordinates - used for scale calculation
     QgsUnitTypes::DistanceUnit mapUnits() const;
 
+    /**
+     * Sets the \a ellipsoid by its acronym. Known ellipsoid acronyms can be
+     * retrieved using QgsEllipsoidUtils::acronyms().
+     * Calculations will only use the ellipsoid if a valid ellipsoid has been set.
+     * \note
```

Comment:
Remove this note - it doesn't belong in the api docs (it's the responsibility of your IDE to work this out on demand!)